packages/itwinui-react/CHANGELOG.md

```diff
@@ -4,10 +4,10 @@
 
 ### Patch Changes
 
+- [#2442](https://github.com/iTwin/iTwinUI/pull/2442): Removed a leftover `@layer` name (`itwinui-v1`), which was originally intended to support _very_ old versions of iTwinUI.
```

Comment:
I intentionally left out the exact layer name from this changeset, because it is an implementation detail of an old version.